src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { format, isValid } from 'date-fns';
+import { connect } from 'react-redux';
+
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import { focusElement } from 'platform/utilities/ui';
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('h2');
+    scrollToTop('topScrollElement');
+  }
+
+  render() {
+    const { form } = this.props;
+    const { formId, data, submission } = form;
+    const submitDate = new Date(submission?.timestamp);
+
+    const { fullName } = data;
+
+    return (
+      <div>
+        <div className="print-only">
+          <img
+            src="https://www.va.gov/img/design/logo/logo-black-and-white.png"
+            alt="VA logo"
+            width="300"
+          />
+          <h2>Application for Mock Form</h2>
+        </div>
+        <h2 className="vads-u-font-size--h3">
+          Your application has been submitted
+        </h2>
+        <p>We may contact you for more information or documents.</p>
+        <p className="screen-only">Please print this page for your records.</p>
+        <div className="inset">
+          <h3 className="vads-u-margin-top--0 vads-u-font-size--h4">
+            22-10215 Statement of Assurance of Compliance with 95 Percent
+            Enrollment Ratios Claim{' '}
+            <span className="vads-u-font-weight--normal">(Form {formId})</span>
+          </h3>
+          {fullName ? (
+            <span>
+              for {fullName.first} {fullName.middle} {fullName.last}
+              {fullName.suffix ? `, ${fullName.suffix}` : null}
+            </span>
+          ) : null}
+
+          {isValid(submitDate) ? (
+            <p>
+              <strong>Date submitted</strong>
+              <br />
+              <span>{format(submitDate, 'MMMM d, yyyy')}</span>
+            </p>
+          ) : null}
+          <button
```

Comment:
Is there a reason for not using va-button here?

Reply:
@powellkerry this is boilerplate code. When you run yarn new:app this is the code that is generated for new forms. This is the initial push for this code. It will live in staging until a collaboration cycle is completed. There will be several parts of this code base that is changed before it sees prod.

Reply:
This has been updated; yeoman generator has other outdated code such as `OMBInfo` vs `va-omb-info` just fyi